apps/organisations/mixins.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from apps.organisations.models import OrganisationTermsOfUse
+
+ORGANISATION_TERMS_OF_USE_LABEL = _(
+    'Yes, I have read and agree to the terms of use. '
```

Comment:
The label probably has to go to the __init__, so that we can actually add the link to the organisation's terms of use.